src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -96,6 +97,8 @@ static AttachmentsStorage()
             });
         }
 
+        private long _missingAttachmentsCount;
```

Comment:
lets expose this to snmp
similar to
https://github.com/ravendb/ravendb/blob/v5.4/src/Raven.Server/Monitoring/Snmp/Objects/Database/5.2.X/1/DatabaseCountOfAttachments.cs

Reply:
done